replica/tablets.cc

```diff
@@ -73,6 +74,7 @@ schema_ptr make_tablets_schema() {
             .with_column("repair_time", timestamp_type)
             .with_column("repair_task_info", tablet_task_info_type)
             .with_column("repair_scheduler_config", repair_scheduler_config_type, column_kind::static_column)
+            .with_column("migration_task_info", tablet_task_info_type)
```

Comment:
Please extend the save/load tests in boost/test_tablets.cc with this new column as well, to ensure we save and load it.